services/controlplane.go

```diff
@@ -39,6 +46,76 @@ func RunControlPlane(ctx context.Context, controlHosts []*hosts.Host, localConnD
 	return nil
 }
 
+func UpgradeControlPlane(ctx context.Context, kubeClient *kubernetes.Clientset, controlHosts []*hosts.Host, localConnDialerFactory hosts.DialerFactory, prsMap map[string]v3.PrivateRegistry, cpNodePlanMap map[string]v3.RKEConfigNodePlan, updateWorkersOnly bool, alpineImage string, certMap map[string]pki.CertificatePKI, upgradeStrategy *v3.NodeUpgradeStrategy, newHosts map[string]bool) error {
+	if updateWorkersOnly {
```

Comment:
Can we add a log line similar to https://github.com/rancher/rke/pull/1800/files#diff-ee57d1a642f3b4ee28fb4929afe13026R55

Reply:
There is this line below it `log.Infof(ctx, "[%s] Processing control plane components for upgrade one at a time", ControlRole)`, that should be good?